plugins/services/src/js/components/ConfigurationMapBooleanValue.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import ConfigurationMapValue from '../../../../../src/js/components/ConfigurationMapValue';
+
+/**
+ * Render a boolean value as a <ConfigurationMapValue>, with it's values being
+ * selected from an `options` array.
+ */
+class ConfigurationMapBooleanValue extends React.Component {
+  render() {
+    return (
+      <ConfigurationMapValue>
+        {this.props.value ? this.props.options[0] : this.props.options[1]}
+      </ConfigurationMapValue>
+    );
+  }
+};
+
+ConfigurationMapBooleanValue.defaultProps = {
+  options: ['Enabled', 'Disabled'],
+  value: false
+};
+
+ConfigurationMapBooleanValue.propTypes = {
+  options: React.PropTypes.array,
+  value: React.PropTypes.any
```

Comment:
I left it `any` in order to be able to evaluate any kind of property in a boolean fashion.